File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/01. Easter Lunch/01. Easter Lunch.js
function easterLunch(EasterCakes, egsPackets, cookiesKg) {
    EasterCakes *= 3.20
    cookiesKg *= 5.40
    let egsPaintingPrice = egsPackets * 12 * 0.15
    egsPackets *= 4.35
    let totalPrice = EasterCakes + egsPackets + cookiesKg + egsPaintingPrice

    return totalPrice.toFixed(2)
}

//console.log(easterLunch(3, 2, 3))

File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/01. Easter Lunch/01. Easter Lunch.py
Easter_cakes = float(input()) * 3.20
egs_packets = float(input())
cookies_kg = float(input())  * 5.40

egs_painting_price =  egs_packets * 12 * 0.15
egs_packets *= 4.35
total_price = Easter_cakes + egs_packets + cookies_kg + egs_painting_price

print(f'{total_price:.2f}')


